src/FloatingActionButton/FloatingActionButton.js

```diff
@@ -166,6 +166,7 @@ class FloatingActionButton extends Component {
   componentWillReceiveProps(nextProps) {
     if (nextProps.disabled !== this.props.disabled) {
       this.setState({
+        hovered: nextProps.disabled ? false : this.state.hovered,
```

Comment:
Well, that's really an issue if the `FloatingActionButton` call the parent component that trigger another render of the `FloatingActionButton`, kind of a loop.